src/conn/conn_api.c

```diff
@@ -3043,6 +3043,12 @@ wiredtiger_open(const char *home, WT_EVENT_HANDLER *event_handler, const char *c
 #endif
 
 err:
+
+#ifdef HAVE_CALL_LOG
+    /* Print the call log entry for wiredtiger_open. */
+    WT_RET(__wt_call_log_wiredtiger_open(session, ret));
```

Comment:
I think we should split this function into two functions. one is called at the beginning (containing the input parameters) and the other one at the end (containing the return values/errors).

Reply:
If that's the case, we should set up the call log earlier in wiredtiger open. Do you have any suggestions for where that should be done?

Reply:
I think no need to split for wiredtiger_open, but I think it is required for other functions. for wiredtiger_open, in case of error, no point in verifying or running that call log through the simulator as we never expect wiredtiger_open to fail on the simulator.